website_sale_menu_partner_top_selling/readme/USAGE.rst

```diff
@@ -0,0 +1,6 @@
+Regular products can be viewed by accessing the URL /shop/my_regular_products on the
+website.
+This URL can be accessed from */shop* by clicking on *My regular products* in the
```

Comment:
RST syntax forces to have a blank link in between paragraphs. You can check that is not working right in https://github.com/OCA/e-commerce/blob/8cbf5ba0d5c7eaabd6c36c99c49813569b16affd/website_sale_menu_partner_top_selling/README.rst

```suggestion

This URL can be accessed from */shop* by clicking on *My regular products* in the
```

(and the same for the rest)

Reply:
Done, thanks